data/src/test/java/org/apache/iceberg/data/TestDataFileIndexStatsFilters.java

```diff
@@ -163,10 +163,9 @@ public void testPositionDeletePlanningPathFilter() throws IOException {
       tasks = Lists.newArrayList(tasksIterable);
     }
 
-    Assert.assertEquals("Should produce one task", 1, tasks.size());
+    Assertions.assertThat(tasks).as("Should produce one task").hasSize(1);
```

Comment:
this is correct. Please make sure that other assertions are updated to use the same. See also my other comment on this